src/components/results/Results.vue

```diff
@@ -1,5 +1,7 @@
 <template>
     <div class="hawk-results" @click="onClick">
+        <autocorrect-suggestions v-if="searchOutput && searchOutput.DidYouMean && searchOutput.DidYouMean.length !== 0" />
```

Comment:
Migrate the v-if condition to the component's template. Will keep the logic encapsulated.